src/thread_task.rs

```diff
@@ -55,13 +53,16 @@ use std::process::abort;
 use std::sync::atomic::AtomicI32;
 use std::sync::atomic::AtomicU32;
 use std::sync::atomic::Ordering;
+use std::sync::MutexGuard;
 
 #[cfg(target_os = "linux")]
 use libc::prctl;
 
 #[cfg(target_os = "macos")]
 use libc::pthread_setname_np;
 
+use super::internal::TaskThreadData_inner;
```

Comment:
Can you use `"rust-analyzer.imports.prefix": "crate"` (and `"rust-analyzer.imports.granularity.group": "item"`) for imports?  I've been avoiding `use super` (except for a few `use super::*`s for inline `mod`s) and also grouping all of the imports together in one block so they're all sorted together.

Reply:
Thanks, didn't notice this. Those are helpful settings.